config/config.go

```diff
@@ -132,6 +132,9 @@ func GetConfigurationValue(config *Config, key string) (string, error) {
 		return config.Author, nil
 	} else if lowerKey == "cpp.lib.location" {
 		return config.CppLibraryLocation, nil
+	} else if strings.HasPrefix(key,"config.templates") {
+    lang := key[strings.LastIndex(key,".") + 1:]
+    return config.CustomTemplate[lang], nil
```

Comment:
Adding a test for it will be awesome.

Reply:
test in `config/config_test.go` right ?

Reply:
Yes the one associated with this config file ie this one : https://github.com/chermehdi/go-egor/blob/master/commands/config_test.go#L11

There is no get test for now tho...